src/components/ListItem.css

```diff
@@ -1,4 +1,4 @@
-.ListItem {
+/* .ListItem {
```

Comment:
If we're not using these styles anymore, can we delete them? (If you end up needing them again, you can always go back in the version history!)